doc/classes/XRInterface.xml

```diff
@@ -97,6 +97,31 @@
 				[param tracker_name] is optional and can be used to direct the pulse to a specific device provided that device is bound to this haptic.
 			</description>
 		</method>
+		<method name="is_passthrough_supported">
+			<return type="bool" />
+			<description>
+				Is [code]true[/code] if this interface supports passthrough.
+			</description>
+		</method>
+		<method name="is_passthrough_supported">
```

Comment:
This should be `is_passthrough_enabled`.

Reply:
Hmm strange that would happen. `godot --doctool .` should update the documentation automatically with the correct name.

Reply:
@m4gr3d @BastiaanOlij Sorry, I missed the part in the development documentation regarding automatic docs update, so I just manually added the new method references. Oops :)

By the way, there seems to be a typo in the [documentation](https://docs.godotengine.org/en/stable/development/cpp/custom_modules_in_cpp.html#writing-custom-documentation):
`user@host:~/godot/bin$ ./bin/<godot_binary> --doctool .`
Here the cwd is already `godot/bin`, though the relative path of the godot binary also contains the `bin` folder.

Reply:
oops, it designed to run from the root of the source folder, you can put a PR in for the documentation on https://github.com/godotengine/godot-docs/ if you want :)

Reply:
I created a PR for the documentation fix: https://github.com/godotengine/godot-docs/pull/6205